src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -101,14 +106,116 @@ def report_initial_status(
         )
 
 
+def append_perf_data_to_central_logs(perf_df: pd.DataFrame, log_path: Path) -> None:
+    """Append performance data to the central logs."""
+    NUM_ROWS_PER_FILE = 100_000
+
+    if not fnmatch(log_path, "/mnt/team/simulation_science/pub/models/*/results/*"):
```

Comment:
I've added so we can be more confident that we're pulling the info from the output directory as expected. Thoughts?